afinn/data/AFINN-no-165.txt

```diff
@@ -0,0 +1,3214 @@
+forsterke	2
+styrking	2
+styrker	2
+understreke	-1
+stresset	-2
+streik	-1
+sterk	2
+sterkere	2
+sterkeste	2
+streve	-2
+slet	-2
+sliter	-2
+sittende fast	-2
+slående	4
+dumhet	-3
+beleven	2
+stevning	-2
+betydelig	1
+i det vesentlige	1
+undergravende	-2
+lykkes	3
+lyktes	3
+suksess	2
+vellykket	3
+suge	-3
+suger	-3
+saksøke	-2
+saksøkt	-2
+sues	-2
+lide	-2
+led	-2
+lidende	-2
+lider	-2
+selvmord	-2
+velegnet	2
+mutt	-2
+solskinn	2
+suveren	5
+overlegen	2
+brukerstøtte	2
+støttet	2
+støttespillere	1
+bære	1
+støttende	2
+støtter	2
+overlevde	2
+overlevende	2
+mistenkt	-1
+mistanke	-1
+mistenkte	-1
+henge	-1
+suspendert	-1
+mistenkelig	-2
+bærekraft	1
+bærekraftig	2
+sverge	-2
+banning	-2
+sverger	-2
+søtere	3
+søteste	3
+fort	2
+raskt	2
+svindle	-3
+svindlerier	-3
+sympati	2
+plett	-2
+skjemt	-2
+tard	-2
+anløpe	-2
+anløpet	-2
+oksiderer	-2
+tårer	-2
+mør	2
+ømhet	2
+anspent	-2
+spenning	-1
+veldig bra	4
+livredd	-3
+skrekk	-3
+terrorister	-2
+terror	-3
+terrorisert	-3
+terroriserer	-3
+takke	2
+takk	2
+tornete	-2
+omtenksom	2
+true	-2
+truer	-2
+begeistret	5
+forpurret	-2
+kneble	-2
+toftene	-2
+engstelige	-2
+trett	-2
+puppene	-2
+toleranse	2
+tannløs	-2
+topp	2
+topper	2
+revet	-2
+tortur	-4
+torturert	-4
+torturere	-4
+totalitær	-2
+totalitarisme	-2
+spioner	-2
+giftig	-3
+tragedier	-2
+tragedie	-2
+tragisk	-2
+overtre	-2
+trådt	-2
+felle	-1
+fanget	-2
+feller	-1
+traume	-3
+traumatisk	-3
+parodi	-2
+forræderisk	-3
+skatt	2
+skatter	2
+skjelving	-2
+skjelvinger	-2
+hyllest	2
+lureri	-2
+triumf	4
+triumferende	4
+problema	-2
+ekte	2
+tillit	1
+klarert	2
+stiftelser	1
+svulst	-2
+tyran	-3
+tyranniske	-3
+tyrannisk	-3
+tyrans	-3
+allestedsnærværende	2
+stygghet	-3
+stygg	-3
+ute av stand	-2
+uakseptabelt	-2
+uattraktiv	-2
+utilgjengelig	-1
+uutholdelig	-2
+ikke til å tro	-1
+objektivt	2
+uklar	-2
+ubekreftet	-1
+usikre	-2
+kokt	-2
+undervurder	-1
+undervurdert	-1
+underestimerer	-1
+underslår	-1
+undergrave	-2
+undergraves	-2
+undergraver	-2
+tapte terreng	-2
+dårlige	-2
+dårlige resultater	-2
+ufortjente	-2
+uønsket	-2
+arbeidsledig	-1
+arbeidsledighet	-2
+ulik	-1
+livskvalitet	2
+uetisk	-2
+begivenhetsløs	-2
+urettferdig	-2
+ugunstig	-2
+uegnet	-2
+uskikket	-2
+ufokusert	-2
+uforsonlig	-2
+utakknemlig	-3
+ulykkelig	-2
+ulykkelighet	-2
+usunn	-2
+uhygienisk	-2
+enhetlig	1
+fantasiløs	-2
+uinspirert	-2
+uintelligent	-2
+forent	1
+lite sannsynlig	-1
+uelsket	-2
+uovertruffen	1
+umotiverte	-2
+uoriginal	-2
+uprofesjonelt	-2
+rakne	1
+uvitenskapelig	-2
+usikret	-2
+uselvisk	2
+uoppgjorte	-1
+usolgt	-1
+usofistikert	-2
+uholdbar	-2
+ustabil	-2
+ustoppelig	2
+mislykket	-2
+uten hell	-2
+støttes	-2
+ubesudlede	2
+usann	-2
+uverdig	-2
+oppløftende	2
+besvær	-2
+som haster	-1
+nyttig	2
+nytten	2
+ubrukelig	-2
+ubrukelighet	-2
+validere	1
+validert	1
+validerer	1
+kjennelse	-1
+dommer	-1
+egen	1
+levende	3
+offer	-3
+ofrene	-3
+seier	3
+seire	3
+årvåken	3
+vigør	3
+sjofel	-3
+stadfester	2
+bryte	-2
+krenket	-2
+bryter med	-2
+vold	-3
+vold relatert	-3
```

Comment:
Why "vold relatert" when "vold" is there?

Reply:
Should it be one word?

Reply:
this is very rarely used so I'll remove it